### PR TITLE
dask: Deal with TODODASKMSG placeholders

### PR DESCRIPTION
**Pull request by davidhassell**
Deal with TODODASKMSG placeholders.

### CODE REVIEW

File: cf/data/mixin/deprecations.py [hunk 1]
```diff
@@ -731,7 +748,11 @@ def partition_boundaries(self):
 
         """
         _DEPRECATION_ERROR_METHOD(
-            "TODODASKMSG - consider using 'chunks' instead"
+            self,
+            "partition_boundaries",
+            message="Consider using the 'chunks' attribute instead.",
```
Comment (sadielbartholomew) [inline cf/data/mixin/deprecations.py:753]: Perhaps, to use the same tone/format as (most, codebase-wide) of the other messages, just:

```suggestion
            message="Use the 'chunks' attribute instead.",
```

?

**Review by sadielbartholomew — APPROVED** [review]
One minor suggestion raised, but all good so please merge when ready.

File: cf/data/mixin/deprecations.py (hunk 1 above)
Comment (davidhassell) [reply cf/data/mixin/deprecations.py:753]: Hi Sadie, I see what you mean, but I try to use "consider" when the suggested alternative doesn't quite do what the deprecated method did, i.e. you can't just change the name and carry on!

Comment (sadielbartholomew) [reply cf/data/mixin/deprecations.py:753]: Ah, that makes sense! Noted for future. Thanks, marking as resolved.

**Pull request merged**